### DISCUSSION TITLE
How this watcherMail work

### DISCUSSION THREAD

**Discussion by abdelrahman-abied** [talsec/Free-RASP-Flutter#52] [Q&A] [answered]

**Comment by msikyna** [comment, marked as answer]
Hello @abdelrahman-abied ,

first, you put a mail address into the **watcherMail** parameter in **TalsecConfig**. Talsec registers this mail address and sends a regular (weekly) security reports describing the application's security state and characteristics of the devices it runs on, e.g. how many devices are rooted or if your application has been tampered. See the [Security Report](https://github.com/talsec/Free-RASP-Flutter#security-report) chapter for more details.

Kind regards,
Matúš, Talsec developer

**Reply by rmazur** [reply]
Hello Matúš,
If I want to test it from my simulator, to check if it works, how can I check? Do I have to wait for 1 week to get a report or is there another way to verify if my implementation works on a simulator or emulator?

Best regards.
Regis
